internal/db/migrations/005_reservations.sql

```diff
@@ -4,7 +4,10 @@ CREATE TABLE reservations
   provider INTEGER NOT NULL CHECK (valid_provider(provider)),
   account_id BIGINT NOT NULL REFERENCES accounts(id),
   created_at TIMESTAMP NOT NULL DEFAULT current_timestamp,
+  steps SMALLINT NOT NULL,
+  step SMALLINT NOT NULL,
```

Comment:
Shall we default to 0?
```suggestion
  step SMALLINT NOT NULL DEFAULT 0,
```